bin/propolis-utils/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "propolis-utils"
+version = "0.0.0"
+authors = ["Patrick Mooney <pmooney@oxide.computer>"]
+license = "MPL-2.0"
+edition = "2018"
```

Comment:
2021 should be fine here, our current MSRV is 1.64 which is new enough (2021 edition was introduced with 1.56).

Reply:
Will change that.  I mostly just cargo-culted when copying stuff over.